examples/postgres/vagga.yaml

```diff
@@ -6,31 +6,76 @@ containers:
   ubuntu:
     setup:
     - !Ubuntu xenial
-    - !Install
-      - postgresql-9.5
+    # Use fixed user id and group id for postgres, because in some cases
+    # we may need to upgrade (rebuild) a postgres container, but keep the data
+    # on a `!Persistent` volume still usable. User ids in ubuntu packages are
+    # not guaranteed to be same on every installation.
+    #
+    # The command-lines are from the postgres-common package except
+    # added --uid 200 --gid 200
+    - !Sh |
+        addgroup --system --gid 200 postgres
```

Comment:
Maybe `adduser`/`addgroup` commands should became `!AddUser` in vagga, shouldn't it?